### PR TITLE
Removed hierarchical registries

### PR DESCRIPTION
**Pull request by piotrrzysko**

### CODE REVIEW

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/MaxRateRegistryPaths.java [hunk 1]
```diff
@@ -6,15 +6,15 @@
 import static pl.allegro.tech.hermes.infrastructure.zookeeper.ZookeeperPaths.MAX_RATE_HISTORY_PATH;
 import static pl.allegro.tech.hermes.infrastructure.zookeeper.ZookeeperPaths.MAX_RATE_PATH;
 
-class FlatBinaryMaxRateRegistryPaths {
+class MaxRateRegistryPaths {
 
     private final static String RATE_BINARY_RUNTIME_PATH = "runtime-bin";
```
Comment (szczygiel-m) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/MaxRateRegistryPaths.java:11]: should't this be renamed to something like `RATE_RUNTIME_PATH`?

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerAssignmentCache.java [hunk 2]
```diff
@@ -1,20 +1,125 @@
...
+            logger.info("Stopping binary workload assignment cache");
+            started = false;
+            workloadNodeCache.close();
+        } catch (IOException e) {
+            throw new RuntimeException("Could not stop node cache for consumer workload", e);
+        }
+    }
+
+    public boolean isStarted() {
+        return started;
```
Comment (szczygiel-m) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerAssignmentCache.java:106]: is this used?

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/WorkloadRegistryPaths.java [hunk 3]
```diff
@@ -4,14 +4,14 @@
 
 import static pl.allegro.tech.hermes.infrastructure.zookeeper.ZookeeperPaths.CONSUMERS_WORKLOAD_PATH;
 
-class FlatBinaryWorkloadRegistryPaths {
+class WorkloadRegistryPaths {
 
     private final static String WORKLOAD_BINARY_RUNTIME_PATH = "runtime-bin";
```
Comment (szczygiel-m) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/WorkloadRegistryPaths.java:9]: same here, shouldn't it be renamed to something like `WORKLOAD_RUNTIME_PATH`?

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerAssignmentCache.java (hunk 2 above)
Comment (piotrrzysko) [reply hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerAssignmentCache.java]: good catch, removed

**Review by szczygiel-m — APPROVED** [review]

**Pull request merged**